clients/source.go

```diff
@@ -307,15 +307,15 @@ func (c *SourceClient) Sync2(ctx context.Context, spec specs.Source, res chan<-
 		Spec: b,
 	})
 	if err != nil {
-		return fmt.Errorf("failed to call Sync: %w", err)
+		return fmt.Errorf("failed to call Sync: %s", extractErrorMessage(err))
 	}
 	for {
 		r, err := stream.Recv()
 		if err != nil {
 			if err == io.EOF {
 				return nil
 			}
-			return fmt.Errorf("failed to fetch resources from stream: %w", err)
```

Comment:
`failed to fetch resources from stream` is incorrect (similar to https://github.com/cloudquery/plugin-sdk/issues/462#issue-1476833829), as error can be anything returned from https://github.com/cloudquery/plugin-sdk/blob/cdcef5d9256ee7d43336e557d1c4db3eda2ee46b/internal/servers/source.go#L93 so not necessarily means something with the stream